containers/app/entrypoint.sh

```diff
@@ -33,8 +33,11 @@ mv /home/opendevin/.cache/ms-playwright/ /home/enduser/.cache/
 
 # get the user group of /var/run/docker.sock and set opendevin to that group
 DOCKER_SOCKET_GID=$(stat -c '%g' /var/run/docker.sock)
-echo "Docker socket group id: $DOCKER_SOCKET_GID"
-usermod -aG $DOCKER_SOCKET_GID enduser
+echo "Docker socket group id: $DOCKER_SOCKET_GID."
+echo "Creating docker group with fetched gid."
+groupadd -g $DOCKER_SOCKET_GID docker
```

Comment:
do we need to check if the group exists first?

Reply:
yes, but this won't break the flow.

Reply:
yes, but it doesn't affect the flow.